examples/kubernetes/README.md

```diff
@@ -19,11 +33,13 @@ In case of any issue, you can reset the cluster from this menu and the button `R
 helm repo add traefik https://traefik.github.io/charts
 helm repo update
 kubectl create ns traefik
-helm upgrade --install --namespace=traefik \
+helm upgrade --version v26.1.0 --install --namespace=traefik \
```

Comment:
Sure, it's just the last one no ?

Reply:
what is just the last one ?

Reply:
I believe it's better to fix the version, so when something might break on Traefik helm side, this will still work with the values.yml